### ISSUE TITLE
Multi-gpu training

### ISSUE THREAD

**Issue by KKN18** [pixeli99/SVD_Xtend#24]
First of, thank you for providing SVD training code.

I'm trying to train using multi-GPU, are there any changes I need to make to the code and prompt? 
In the code, should I uncomment this part and what else do I need to do?
```
    # ddp_kwargs = DistributedDataParallelKwargs(find_unused_parameters=True)
    accelerator = Accelerator(
        gradient_accumulation_steps=args.gradient_accumulation_steps,
        mixed_precision=args.mixed_precision,
        log_with=args.report_to,
        project_config=accelerator_project_config,
        # kwargs_handlers=[ddp_kwargs]
    )
```

**Comment by KKN18** [comment]
Oh I see. Thank you!

**Comment by chenbinghui1** [comment]
@pixeli99 hello, I try to use multiple-gpus, while the following error shows up:
![image](https://github.com/pixeli99/SVD_Xtend/assets/11517207/84aec31f-0fe6-4b2a-a2dc-c8656bf5f6b9)

The command is like:
`CUDA_VISIBLE_DEVICES=0,1 accelerate launch ashui_train_svd.py     --pretrained_model_name_or_path="models/svd"     --pretrain_unet="models/svd_unet_11channels"     --gradient_checkpointing     --per_gpu_batch_size=1 --gradient_accumulation_steps=2     --max_train_steps=400000     --num_frames=25     --width=512     --height=896     --checkpointing_steps=10000 --checkpoints_total_limit=100     --learning_rate=1e-5 --lr_warmup_steps=0     --seed=42     --mixed_precision="fp16"     --validation_steps=200000     --output_dir="./outputs/svd"`


while, when only using one gpu like CUDA_VISIBLE_DEVICES=0, the training goes correct.
So do I miss something?
